src/SmartComponents/HybridInventoryTabs/HybridInventoryTabs.js

```diff
@@ -42,7 +42,13 @@ const HybridInventory = ({
       appName="inventory"
       module="./HybridInventoryTabs"
       ConventionalSystemsTab={
-        <Suspense fallback={Fragment}>
+        <Suspense
+          fallback={
+            <Bullseye>
+              <Spinner size="lg" />
```

Comment:
Let's make spinners for `ConventionalSystemsTab` and `ImmutableDevicesTab` the same size